package.json

```diff
@@ -19,7 +19,7 @@
   "author": "Mikael Brevik",
   "license": "MIT",
   "peerDependencies": {
-    "marked": "^1.0.0"
+    "marked": "^1.0.0 || ^2.0.0"
```

Comment:
Why do you want to support `marked@1`? It has some [vulnerabilities](https://github.com/advisories/GHSA-4r62-v4vq-hr96)

Reply:
This package still works with v1. If someone needs to use v1 for whatever reasons they should not be prevented from updating this package.